index.js

```diff
@@ -267,6 +268,12 @@ function onReady () {
             case consts.eventNames.windowCmdShow:
             elements[json.targetID].show()
             break;
+            case consts.eventNames.windowCmdLoadUrl:
+            elements[json.targetID].loadURL(json.url, {}).then(() => {
```

Comment:
Can you confirm that `astilectron` JS namespace is available in the loaded URL ?

Namely is [this logic](https://github.com/asticode/astilectron/blob/master/index.js#L449) still called when loading the URL here ?

Reply:
I can confirm it is available.